package/tests/integration/inputs/path/duplicate-path/Nickel-pkg.ncl

```diff
@@ -0,0 +1,11 @@
+{
+  name = "duplicate-path",
+  description = "A package having two differently named dependency paths to the same path package",
+  version = "0.1.0",
+  authors = ["Joe"],
+  minimal_nickel_version = "1.9.0",
+  dependencies = {
+    dep = 'Path "../single-path-dep",
+    leafy = 'Path "../leaf",
```

Comment:
Shouldn't the paths be the same here? That what's I thought from the description.

Reply:
"dep" also has `'Path "../leaf"` as a dependency. I changed the description to "A package having two different routes in the dependency graph to the same path package". Is that clearer?